### PR TITLE
Add Hide/show desktop icon switch

### PR DESCRIPTION
**Pull request by cinit**
# 添加桌面图标开关

## 介绍 / Description
在主界面添加了桌面图标开关

## 更改内容 / Types of Changes
- [x] 修复 Bugfix
- [x] 增强/优化 Enhancement/optimization

## 由这个PR解决或关闭的Issues / Issues Fixed or Closed by This PR
-  #418 

## 检查列表 / Checklist
- [x] 我的代码符合本仓库的代码规范 My code follows the code style of this project
- [x] 我已经测试了这些更改，它们可以正常工作，且不会破坏任何功能(或我可以解决) I have tested the changes and verified that they work and don't break anything (as well as I can manage).
- [x] 我已合并了对后续工作无意义的commit并确认不会对后续维护造成困扰 I have merged commits that are meaningless for follow-up work and confirmed that they will not cause damage to follow-up maintenance

### CODE REVIEW

**Review by Lagrio — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/main_v2.xml [hunk 1]
```diff
@@ -12,14 +12,48 @@
...
+                android:textSize="16sp" />
+
+            <com.google.android.material.switchmaterial.SwitchMaterial
+                android:id="@+id/mainV2_showLauncherIconSwitch"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content" />
+
+        </LinearLayout>
+
+    </LinearLayout>
```
[the remark this answers is not in the document]
Comment (cinit) [reply app/src/main/res/layout/main_v2.xml:55]: 菜单里怎么放Switch?

[the remark this answers is not in the document]
Comment (Lagrio) [reply app/src/main/res/layout/main_v2.xml:55]: 放什么switch，不需要switch

[the remark this answers is not in the document]
Comment (cinit) [reply app/src/main/res/layout/main_v2.xml:55]: 两个按钮不是很蠢吗

[the remark this answers is not in the document]
Comment (Lagrio) [reply app/src/main/res/layout/main_v2.xml:55]: 你难道还想_隐藏_和_显示_一起放不成？

**Pull request closed without merging**